cmd/archer/template/version.go

```diff
@@ -4,4 +4,5 @@
 // Package template provides usage templates to render help menus.
 package template
 
-const VersionFlag = `Archer version: {{.Version}}`
+// VersionFlag is the template string used for the --version flag output.
+const VersionFlag = "Archer version: {{.Version}}\n"
```

Comment:
nit: sorry I know this is not part of your PR, but what do you think of renaming this const to just `Version`?

so that it reads `template.Version`?

Reply:
We could also just inline the string altogether and drop this file? it's currently only used in one place.

Reply:
up to you, I don't mind having all the templates in a single pkg :)